cmd/configrepo/preflight.go

```diff
@@ -48,9 +47,13 @@ func (pr *PreflightRunner) handleApiResponse(res *dub.Response) error {
 		if b, err := ioutil.ReadAll(reader); err != nil {
 			return err
 		} else {
+			if res.IsAuthError() {
+				utils.Die(1, `Invalid credentials. Either the username or password configured is incorrect`)
```

Comment:
Here, I would use `utils.DieLoudly()` as this will print the error to output even if the `--quiet` flag is set. That's largely the difference between `Die()` and `DieLoudly()`; one honors `--quiet` and the other does not.